src/app/pipes/linkColumn.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'linkColumn'
+})
+export class LinkColumnPipe implements PipeTransform {
+
+  transform(item: any): any {
```

Comment:
It might be simpler if this pipe accepted the same arguments as `TableColumnPipe` and then would return `null` if there is no link, `'single'` if it's a single link, and `'multiple'` if it's a multiple link type.

This would allow you to simplify the Angular template with an ngSwitch with the following cases:
* if single
* if multiple
* else there is no link

Reply:
If we are doing this do we still want to use the new `linkColumnMappings` format we came up with?